src/index.js

```diff
@@ -0,0 +1,44 @@
+import './style.css';
+
+const task0 = {
+  description: 'wash the dishes',
+  completed: false,
+  index: 0,
+};
+
+const task1 = {
+  description: 'complete To Do list project',
+  completed: false,
+  index: 1,
+};
+
+const toDoList = [task0, task1];
```

Comment:
- [ ] [OPTIONAL]  To make the code more readable, I would recommend you to create an array of objects rather than assigning two objects, and then creating an array, e.g: 
```
const toDoList = [
  {
    description: 'wash the dishes',
    completed: false,
    index: 0,
  },
  {
    description: 'complete To Do list project',
    completed: false,
    index: 1,
  },
];
```